robotools/transform.py

```diff
@@ -200,9 +200,28 @@ def __init__(self, original_shape: tuple, random_seed: int) -> None:
         self.original_shape = original_shape
         self.random_seed = random_seed
         self.rng = numpy.random.RandomState(self.random_seed)
-        self.original_wells = make_well_array(*self.original_shape).flatten()
-        self.randomized_wells = self.rng.permutation(self.original_wells).tolist()
-        self.lookup = {owell: rwell for owell, rwell in zip(self.original_wells, self.randomized_wells)}
+        self.lookup: Dict[str, str] = {}
+        full = make_well_array(*self.original_shape)
+        if mode == "full":
+            self.original_wells = full.flatten()
+            self.randomized_wells = []
```

Comment:
```suggestion
```
This line can be removed because the attribute is set/overwritten by the line below